pkg/clients/edgeconnect/client.go

```diff
@@ -149,25 +153,25 @@ func (c *client) getServerResponseData(response *http.Response) ([]byte, error)
 }
 
 // GetEdgeConnect returns edge connect if it exists
-func (c *client) GetEdgeConnect(edgeConnectId string) (GetResponse, error) {
+func (c *client) GetEdgeConnect(edgeConnectId string) (EdgeConnectResponse, error) {
```

Comment:
I intentionally named it short. If you check the package name:
Now we have `edgeconnect.EdgeConnectResponse` instead of `edgeconnect.GetResponse`

Reply:
I wanted to have 
- EdgeConnectResponse
- EdgeConnectsResponse

but IMO
- GetResponse
- ListResponse

you suggest are better because there is no duplication of `EdgeConnect` in names
thx